.github/workflows/live-service-testing.yml

```diff
@@ -0,0 +1,45 @@
+name: Live service testing
+on:
+  schedule:
+    - cron: "0 16 * * *"  # Daily at noon EST
```

Comment:
So call this daily, not with PRs?

Reply:
Like the main tests, it's both - its the 'workflow_call' trigger that gets called from the main deployment that triggers on PRs

Reply:
oh ok